gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystemOptions.java

```diff
@@ -24,30 +24,29 @@ public abstract class GoogleCloudStorageFileSystemOptions {
 
   public static GoogleCloudStorageFileSystemOptions DEFAULT = builder().build();
 
-  public enum FilesystemAPI {
-    OBJECT,
-    DIRECTORY,
-    AUTO
+  /** Cloud Storage client to use. */
+  public enum ClientType {
+    HTTP_API_CLIENT,
+    CLIENT_LIBRARY
```

Comment:
do we want to change this to JAVA_STORAGE_CLIENT or STORAGE_CLIENT to match https://github.com/googleapis/java-storage ?

Reply:
Done